cns/NetworkContainerContract.go

```diff
@@ -233,8 +233,8 @@ type GetIPAddressesRequest struct {
 
 // GetIPAddressStateResponse is used in CNS IPAM mode as a response to get IP address state
```

Comment:
Is this only used for debug API and not any other scenario like ACI/Batch? @matmerr ?